packages/app/src/cli/services/dev/ui/components/Dev.tsx

```diff
@@ -80,6 +83,7 @@ const Dev: FunctionComponent<DevProps> = ({
       }, 2000)
     }
     clearInterval(pollingInterval.current)
+    await app.developerPlatformClient.devSessionDelete({appId: app.id, shopFqdn})
```

Comment:
I see the [Partners client is throwing an error](https://github.com/Shopify/cli/blob/866af380a46018ed1168a0ef15e4cfa36d24f165/packages/app/src/cli/utilities/developer-platform-client/partners-client.ts#L537) for this operation. Shouldn't it just do nothing instead or this is only happening when consistent dev is enabled?